state/memcached/memcached.go

```diff
@@ -15,12 +15,13 @@ import (
 )
 
 const (
-	hosts              = "hosts"
-	maxIdleConnections = "maxIdleConnections"
-	timeout            = "timeout"
+	hosts              			= "hosts"
+	maxIdleConnections 			= "maxIdleConnections"
+	timeout            			= "timeout"
+	ttlInSeconds       			= "ttlInSeconds"
 	// These defaults are already provided by gomemcache
-	defaultMaxIdleConnections = 2
-	defaultTimeout            = 1000 * time.Millisecond
+	defaultMaxIdleConnections	= 2
```

Comment:
Actually it's look normal in my editor (VS Code), but in the PR looks like that. I changed encoding but  It's not resolve the problem. Should I need to try other solutions?

Reply:
```suggestion
	defaultMaxIdleConnections = 2
```

Reply:
You can just accept this and then it should work.